docs/creating-locale-files.md

```diff
@@ -0,0 +1,44 @@
+# Creating locale files
```

Comment:
Should this be renamed to 'Translating locale files', as this guidance is aimed at translating existing locale files?